ws-nextjs-app/pages/[service]/av-embeds/handleAvRoute.tsx

```diff
@@ -15,48 +15,44 @@ export default async (context: GetServerSidePropsContext) => {
     req: { headers: reqHeaders },
   } = context;
 
-  let pageStatus = 200;
+  let pageStatus;
   let pageJson;
 
-  const env = getEnvironment(resolvedUrl);
-  const agent = certsRequired(resolvedUrl) ? await getAgent() : null;
-
   const parsedRoute = parseAvRoute(resolvedUrl);
 
-  const extension = `${parsedRoute.service ?? ''}${parsedRoute.variant ? `/${parsedRoute.variant}` : ''}/${parsedRoute.assetId}`;
+  const avEmbedsUrl = constructPageFetchUrl({
+    pageType: 'avEmbeds',
```

Comment:
Could you use the `AV_EMBEDS` const from pageTypes?

```suggestion
    pageType: AV_EMBEDS,
```